Documentation augmentation (#15)
CitrineInformatics/taurus-documentation: docs/faq.md

One ingredient points to "A" as its `material` and has `layer 1` as its name.
One points to "B" as its `material` and has `layer 2` as its name.
And the final ingredient spec _also_ points to "A" as its `material` but is differentiated because it has `layer 3` as its name.

> ## What are is the difference between a description and notes?

Reviewer: ```suggestion
## What is the difference between `description` and `notes`?
```

`description` is a field on both [Attribute Templates](../specification/attribute-templates) and [Object Templates](../specification/object-templates).

> It is used to describe the type of data that a template is intended to constrain; documentation of intended use.

Reviewer: I'm not sure what the clause after the semicolon is meant to convey

Author: Probably better as a hyphen.

> It is strongly encouraged to document all templates, given both how important they are in constraining data and communicating structure to analysis algorithms, and in how much reuse a template is likely to get by a range of users on a given  platform.

Reviewer: ```suggestion
We strongly encourage documenting all templates, given both how important they are in constraining data and communicating structure to analysis algorithms, and that a template is likely to be reused by multiple users.
```
I removed "on a given platform" because I imagine that for the typical consumer of this document (an end user), they only see one platform (their deployment)

Author: I had platform in there intentionally, since it might not even be a citrine platform -- model independent of the instantiation.  But it's a fair critique, and I'll stick with your change if that's a strong feeling on your part.

Reviewer: That is true, but I still think that the important information is conveyed without the clause.

> `notes` are associated with with [Attributes](../specification/attributes) and [Objects](../specification/objects).

Reviewer: ```suggestion
`notes` are associated with [Attributes](../specification/attributes) and [Objects](../specification/objects).
```

> These are particular pieces of information which may be important in understanding this particular piece of data but do not naturally fit in other fields of these objects.

Reviewer: ```suggestion
This is a place to put pieces of information that may be important to understanding this particular piece of data but do not naturally fit in the other fields.
```

This might include an annotation about something unusual about this particular sample.

> These would normally be information that is useful for a human who might review this record but would not be useful in training a machine learning model.

Reviewer: ```suggestion
Notes normally contain information that is useful for a human but would not be useful in training a machine learning model.
```
